.travis.yml

```diff
@@ -32,19 +32,19 @@ matrix:
   include:
     - jdk: oraclejdk8
       env:
-        - FLINK_VERSION="1.3.0" SCALA_VERSION="2.11"
+        - FLINK_VERSION="1.5.1" SCALA_VERSION="2.11"
         - CACHE_NAME=JDK8_F130_A
     - jdk: oraclejdk8
       env:
-        - FLINK_VERSION="1.3.0" SCALA_VERSION="2.10"
+        - FLINK_VERSION="1.5.1" SCALA_VERSION="2.10"
         - CACHE_NAME=JDK8_F130_B
     - jdk: openjdk8
       env:
-        - FLINK_VERSION="1.3.0" SCALA_VERSION="2.11"
+        - FLINK_VERSION="1.5.1" SCALA_VERSION="2.11"
```

Comment:
the version on scala 2.10 was deprecated on flink version 1.4.0
you have to remove this test

Reply:
I know:) I am switching this to scala 2.12

Reply:
If scala 2.10 is deprecated, I would remove them from the travis profiles (as they are the ones failing) and would submit support for 2.12 as a new PR.

Reply:
Ok, this seems reasonable. I will remove them completely then.